docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Enroll an agent into automatic updates
+description: How to enroll an agent into automatic updates
+---
+
+<Admonition type="warning">
+The instructions vary between Teleport Cloud users and self-hosted
+Teleport users. Make sure to select the right scope (OpenSource, Enterprise, or
+Cloud) in the scope selector at the top of the page.
+</Admonition>
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  Automatic agent update is available starting from Teleport `13.0`.
+</Details>
+
+Starting with Teleport 13, agents running in systemd or Kubernetes environments
+can be automatically updated. The [automatic updates architecture
+page](../../architecture/automatic-agent-update.mdx) describes how agent
+updating works.
+
+This guide explains how to enroll an existing Teleport agent into automatic
+updates.
+
+## Requirements
+
+- a Teleport agent either:
```

Comment:
In this context, each first-level bullet is a sentences, and should be adjusted. For example:

```suggestion
- A Teleport agent, either:
```